rocAL/rocAL/source/pipeline/tensor.cpp

```diff
@@ -212,6 +236,73 @@ void rocalTensor::update_tensor_roi(const std::vector<uint32_t> &width,
             }
         }
     }
+    else if(!_info.is_metadata())
+    {
+        auto max_dims = _info.max_shape();
+        unsigned max_samples = max_dims.at(0);
+        unsigned max_channels = max_dims.at(1);
+
```

Comment:
Too many empty lines in this piece of code

Reply:
removed